packages/kbn-guided-onboarding/src/components/landing_page/use_case_card.tsx

```diff
@@ -84,18 +84,13 @@ export const UseCaseCard = ({
       </h4>
     </EuiText>
   );
-  const descriptionElement = (
```

Comment:
This was causing `<div>` to be a descendent of `<p>` and throwing errors in the jest tests

Reply:
I think there is a minor issue with centering the whole content in the card is that it also centers the progress labels, but maybe it's possible to reverse it with a prop on the progress component? 
<img width="265" alt="Screenshot 2022-10-25 at 19 02 26" src="https://user-images.githubusercontent.com/6585477/197837429-21cff85c-74ce-4402-a870-a7e01ea1e572.png">

Reply:
Ah, I missed that. Nice catch! Should be fixed via https://github.com/elastic/kibana/pull/143808/commits/d86c44c33037cf70c638301b625e950a471e5d13.